### PR TITLE
V14 QA Added acceptance tests for Members and Member Group

### PR DESCRIPTION
**Pull request by nhudinh0309**

### CODE REVIEW

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Members/Members.spec.ts [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    await umbracoUi.member.clickCreateButton();
+    await umbracoUi.member.enterMemberName(memberName);
+    await umbracoUi.member.enterUsername(username);
+    await umbracoUi.member.enterEmail(email);
+    await umbracoUi.member.enterPassword(password);
+    await umbracoUi.member.enterConfirmPassword(password);
+    await umbracoUi.member.clickDetailsTab();
+    await umbracoUi.member.enterComments(comment);
+    await umbracoUi.member.clickSaveButton();
+
```
Comment (andr317c) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Members/Members.spec.ts:39]:
```suggestion
```

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Members/Members.spec.ts [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+  });
+
+  test.afterEach(async ({umbracoApi}) => {
+    await umbracoApi.memberType.ensureNameNotExists(memberTypeName);
+    await umbracoApi.member.ensureNameNotExists(memberName);
+  });
+
+  test('can create a member', {tag: '@smoke'}, async ({umbracoApi, umbracoUi}) => {
+    // Act
+    await umbracoUi.member.goToSection(ConstantHelper.sections.members);
```
Comment (andr317c) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Members/Members.spec.ts:29]: Could be in arrange

File: tests/Umbraco.Tests.AcceptanceTest/tests/auth.setup.ts [hunk 3]
```diff
@@ -5,6 +5,7 @@ import {ConstantHelper, UiHelpers} from "@umbraco/playwright-testhelpers";
 setup('authenticate', async ({page}) => {
   const umbracoUi = new UiHelpers(page);
 
+  await umbracoUi.page.route("**/*.{png,jpg,jpeg,webp}",(route) => route.abort());
```
Comment (andr317c) [inline tests/Umbraco.Tests.AcceptanceTest/tests/auth.setup.ts:8]: Why is this added?

**Review by andr317c — CHANGES_REQUESTED** [review]
Had some comments

File: tests/Umbraco.Tests.AcceptanceTest/tests/auth.setup.ts (hunk 3 above)
Comment (nhudinh0309) [reply tests/Umbraco.Tests.AcceptanceTest/tests/auth.setup.ts:8]: It was here accidentally. Removed it.

**Review by andr317c — APPROVED** [review]
LGTM!

**Pull request merged**